src/loaders/compute_tools/microtrait/microtrait.py

```diff
@@ -47,6 +92,52 @@ def _r_table_to_df(r_table):
     return df
 
 
+def _process_trait_counts(
```

Comment:
I'm assuming here that this does exactly the same thing (including error checking) the parser code does other than the change in the cell IDs? Is that right?

Reply:
yea. I copied error checking as well. There isn't a lot at this stage. The majority of checking happen at the next step where global parser reads in individual rows and build the model object.

Reply:
:+1: